llmfoundry/callbacks/model_gauntlet_callback.py

```diff
@@ -44,7 +43,7 @@ class ModelGauntlet(Callback):
     """
 
     def __init__(self,
-                 logger_keys: dict,
+                 logger_keys: list,
```

Comment:
Since this is an API change, can we add in a check below, e.g.
```
if isinstance(logger_keys, dict):
   raise ValueError('logger_keys now requires a list type as input, not a dict') 
```

The code path now seems to support both, but down the road, people might have silent failures if we start depending on the type being a list instead of a dict.

Reply:
done!